assets/openapi.yaml

```diff
@@ -14,6 +14,34 @@ components:
       type: http
       scheme: basic
   parameters:
+    ActivityType:
+      in: path
+      name: activity_type
+      required: true
+      schema:
+        type: string
+        enum:
+          - FILL
+          - ACATC
+          - ACATS
+          - CSD
+          - CSR
+          - CSW
+          - DIV
+          - DIVCGL
+          - DIVCGS
+          - DIVNRA
+          - DIVROC
+          - DIVTXEX
+          - INT
+          - JNLC
+          - JNLS
+          - MA
+          - NC
+          - PTC
+          - REORG
+          - SSO
+          - SSP
```

Comment:
I think these are not the all of the possible activity types (even though the doc might only have these)
if you look at the code we have more (e.g. CIL). https://github.com/alpacahq/gobroker/blob/6470c149664e25224f4022a59de6b7134830a34e/models/enum/entrytypes.go#L7
Its worth to mention this to a PM to either update the doc or remove the unnecessary from the code